src/rpc/rpcevo.cpp

```diff
@@ -974,13 +974,13 @@ UniValue protx_list(const JSONRPCRequest& request)
 #ifdef ENABLE_WALLET
         LOCK2(cs_main, pwallet->cs_wallet);
 
-        if (request.params.size() > 3) {
+        if (!request.params[3].isNull()) {
```

Comment:
Same as with masternode_list, this should be reverted

Reply:
Actually...this is a bug. This should have been `> 4`, as otherwise it bails out when the height is specified. I'll open a separate PR to fix this